test/unit/legacy_wrappers/admin.test.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const { MongoClient } = require('../../../src/index');
+const { Admin } = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+describe('legacy_wrappers/admin.js', () => {
+  let admin;
+  let client;
+  beforeEach(async function () {
+    client = new MongoClient('mongodb://iLoveJs');
+    admin = client.db().admin();
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  describe('addUser', () => {
+    describe('callbacks', () => {
+      it('should admin.addUser(username, callback)', done => {
```

Comment:
can we similarly improve these `it` statements to be more descriptive of what's actually being tested? clearly indicate success/failure expectations and all that jazz? (see my long comment on api.test.js)

Reply:
I can apply the same organization changes here that I made to the programatic tests but I'll hold off until we agree on that way of organizing things so I can replicate it here and with the other manual tests